src/java/htsjdk/tribble/dbsnp/OldDbSNPCodec.java

```diff
@@ -89,6 +89,11 @@ public OldDbSNPFeature decode(String line) {
     }
 
     @Override
+    public boolean canDecode(String path){
+	return path.toLowerCase().endsWith(".dbsnp") || path.toLowerCase().endsWith(".rod") || path.toLowerCase().endsWith(".snp");
```

Comment:
How will this affect current users?  Does this require them to change all their files to ".dbsnp"?